cmd/detect.go

```diff
@@ -51,15 +51,15 @@ var detectCmd = &cobra.Command{
 func runDetect(cmd *cobra.Command, args []string) {
 	// start timer
 	start := time.Now()
-	source := mustGetStringFlag(cmd, "source")
+	sourcePath := mustGetStringFlag(cmd, "source")
```

Comment:
🙏🏻 every time I opened this file I would change this to `sourcePath` then forget about it. Thank you lol